Makefile

```diff
@@ -37,8 +42,3 @@ test-plugin:
 .PHONY: test-shell
 test-shell:
 	./pants test ::
-
-########################################################################
-
-.PHONY: all
```

Comment:
Did you mean to delete this?

Reply:
Yeah, I moved it to the top of the file.